### PR TITLE
Events

### PR DESCRIPTION
**Pull request by tashisrael**
# In this PR, I implemented the following:
- Copied the files `calculate.js` and `operate.js` into a `logic/` directory in the project.
- Analyzed the files `calculate.js` and `operate.js` and understood what they do.
-  Added [big.js](https://www.npmjs.com/package/big-js) as dependency.
- Refactored the components folder
- Imported the above files in your Calculator component.
- Implemented the event handlers needed to use the math logic from `calculate.js` and `operate.js`.
- Used state and props.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,8 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "big-js": "^3.1.3",
+    "big.js": "^6.2.1",
```
Comment (descholar-ceo) [inline package.json]: - [x] Could you install one version, the latest one could be the recommendable one

File: src/App.js [hunk 2]
```diff
@@ -1,10 +1,31 @@
+/* eslint-disable react/state-in-constructor */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prefer-stateless-function */
```
Comment (descholar-ceo) [inline src/App.js:3]: - [x] Don't disable `eslint rules`

File: src/components/Buttons.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/prefer-stateless-function */
```
Comment (descholar-ceo) [inline src/components/Buttons.js:3]: - [x] Don't disable eslint rules

File: src/components/Calculator.js [hunk 4]
```diff
@@ -1,33 +1,24 @@
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
```
Comment (descholar-ceo) [inline src/components/Calculator.js:3]: - [x] Don't disable eslint rules

**Review by descholar-ceo — APPROVED** [review]
Hi @tashisrael 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Implemented all of the requirements ✔️ 

## Optional
- [ ] Just fix the minor comments I put inline your code

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (tashisrael) [reply package.json]: Resolved. Thank you

**Pull request merged**